Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -966,14 +966,17 @@ def get_incident_extra_data_command(client, args):
     incident = raw_incident.get('incident')
     incident_id = incident.get('incident_id')
     raw_alerts = raw_incident.get('alerts').get('data')
-    alerts = clear_trailing_whitespace(raw_alerts)
+    context_alerts = clear_trailing_whitespace(raw_alerts)
+    for alert in context_alerts:
+        alert['host_ip_list'] = alert.get('host_ip').split(',') if alert.get('host_ip') else []
     file_artifacts = raw_incident.get('file_artifacts').get('data')
     network_artifacts = raw_incident.get('network_artifacts').get('data')
 
     readable_output = [tableToMarkdown('Incident {}'.format(incident_id), incident)]
 
-    if len(alerts) > 0:
-        readable_output.append(tableToMarkdown('Alerts', alerts))
+    if len(context_alerts) > 0:
+        readable_output.append(tableToMarkdown('Alerts', context_alerts,
+                                               headers=[key for key in context_alerts[0] if key != 'host_ip']))
```

Comment:
```suggestion
                                               headers=[key for key in context_alerts[0].keys() if key != 'host_ip']))
```

Reply:
when you do for on dict, you iterates on it's keys by default, without writing .keys() 
so it is working without the suggested change.